plc4j/drivers/ads/src/main/java/org/apache/plc4x/java/ads/AdsPlcDriver.java

```diff
@@ -72,7 +72,7 @@ public String getProtocolName() {
     }
 
     @Override
-    public PlcConnection connect(String url) throws PlcConnectionException {
+    public PlcConnection getConnection(String url) throws PlcConnectionException {
```

Comment:
Why did you rename these? For me there is a difference between a getter and a method that actually connects and returns a connection as a result.